### PR TITLE
Docker Script Fix

### PR DESCRIPTION
**Pull request by saquino0827**
### Information
- QPPCT-661

### Changes proposed in this PR:
- Change docker_deploy.sh to return failure on an ansible failure

### Checklist 
- [n/a] All JUnit tests pass (`mvn clean verify`).
- [n/a] New unit tests written to cover new functionality.
- [n/a] Added and updated JavaDocs for non-test classes and methods.
- [n/a] No local design debt. Do you feel that something is "ugly" after your changes?
- [n/a] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: tools/bin/docker_deploy.sh [hunk 1]
```diff
@@ -16,6 +16,10 @@ function run_ansible() {
 if [[ "$CIRCLE_BRANCH" == "master" || ( ! -z $DOCKER_DEPLOY_OTHER_BRANCH && "$CIRCLE_BRANCH" == "$DOCKER_DEPLOY_OTHER_BRANCH" ) ]]; then
 	build_and_tar_converter_image
 	run_ansible ${DOCKER_DEPLOY_HOSTS} ${DOCKER_DEPLOY_REMOTE_USERNAME} ${DOCKER_DEPLOY_SUDOER_PASSWORD}
+	if [[ $? -ne 0 ]]; then
```
Comment (saquino0827) [inline tools/bin/docker_deploy.sh]: My spaces aren't incorrect??

Comment (saquino0827) [reply tools/bin/docker_deploy.sh]: Much better

**Comment by omnivion** [comment]
Can you explain the changes a bit more to the less-than-proficient shell scripters (me)?

**Comment by saquino0827** [comment]
@omnivion sure. It is checking ansible's response to see if it is a non-zero response and purposefully exiting with -1. Before, we were getting the failure response from ansible but then just ignoring it and continuing to run.

**Pull request merged**